ext/tiny_tds/extconsts.rb

```diff
@@ -2,7 +2,7 @@
 ICONV_VERSION = ENV['TINYTDS_ICONV_VERSION'] || "1.15"
 ICONV_SOURCE_URI = "http://ftp.gnu.org/pub/gnu/libiconv/libiconv-#{ICONV_VERSION}.tar.gz"
 
-OPENSSL_VERSION = ENV['TINYTDS_OPENSSL_VERSION'] || '1.1.1d'
+OPENSSL_VERSION = ENV['TINYTDS_OPENSSL_VERSION'] || '1.1.1s'
```

Comment:
I assume this is fine. Are the 1.1.1 release variants cross compatible with each other? Is there any downside to moving as far forward as the latest `l` (L) release?

Reply:
FYI, `l` doesn't refer to latest, they're just going through the alphabet on releases for `1.1.1`, and [`s` is the most recent](https://www.openssl.org/news/cl111.txt) at the moment